site/_posts/2021-02-19-octant-0.17-release.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Kubernetes: There is an app for that"
+image: /img/posts/2020/11/19/survey-results-banner.png
+excerpt: With release 0.17, Octant is available as a desktop application fully packed with new features: improved Navigation, Breadcrumbs, Preferences and much more. 
```

Comment:
Error: YAML Exception reading /home/sfoo/go/src/github.com/vmware-tanzu/octant/site/_posts/2021-02-19-octant-0.17-release.md: (<unknown>): mapping values are not allowed in this context at line 4 column 104

Reply:
Oops, sorry for that, quoted the string